db/c.cc

```diff
@@ -527,6 +531,32 @@ rocksdb_t* rocksdb_open_for_read_only(
   return result;
 }
 
+rocksdb_backup_engine_t* rocksdb_backup_engine_open(
+    const char* path,
+    char** errptr) {
+  BackupEngine* be;
+  if (SaveError(errptr, BackupEngine::Open(Env::Default(), BackupableDBOptions(path), &be))) {
```

Comment:
Can we also make Env* configurable? Clients might want to pass HdfsEnv here, effectively backing up RocksDB to HDFS.